SLURM_PHOENIX_README.md

```diff
@@ -0,0 +1,28 @@
+# Running Conformance Tests on the Phoenix SLURM Cluster
```

Comment:
Slurm is [just a word apparently](https://slurm.schedmd.com/overview.html), not an acronym, so we probably shouldn't call it SLURM everywhere.